src/compiler/tracing.ts

```diff
@@ -145,14 +145,15 @@ namespace ts { // eslint-disable-line one-namespace-per-file
         }
         // sample every 10ms
         const sampleInterval = 1000 * 10;
-        function writeStackEvent(index: number, endTime: number) {
+        function writeStackEvent(index: number, endTime: number, results?: Args) {
             const { phase, name, args, time, separateBeginAndEnd } = eventStack[index];
+            const combinedArgs = { ...args, ...results };

```

Comment:
I might be overthinking, but I wonder if `{ ...args, results }` might be preferable, so that results are grouped and args don't get clobbered.